.remarkrc.mjs

```diff
@@ -1,11 +1,20 @@
 import remarkValidateLinks from 'remark-validate-links';
 import remarkFrontmatter from 'remark-frontmatter';
 import remarkLintFrontmatterSchema from 'remark-lint-frontmatter-schema';
+import remarkLintNoDeadUrls from 'remark-lint-no-dead-urls'
 
 const remarkConfig = {
 	plugins: [
 		remarkValidateLinks,
 		remarkFrontmatter,
+		[
```

Comment:
It probably doesn't matter, but for readability it seems like the new `remarkLintNoDeadUrls` config should come after the frontmatter-related configs so that the frontmatter configs stay grouped directly under the frontmatter plugin.